root/static/lib/jquery.ui/ui/jquery-ui.custom.js

```diff
@@ -5055,9 +5055,6 @@ $.widget( "ui.autocomplete", {
 	},
 
 	__response: function( content ) {
-		if ( content ) {
```

Comment:
I don't mind making changes to this file because jQuery UI is deprecated in our codebase (new components just use React), so we don't plan on upgrading it.

Reply:
We do the same with knockout already, for similar reasons: https://github.com/mwiencek/knockout/commits/mb (new uses of knockout are deprecated, and we are stuck on a commit in between releases and were never able to upgrade due to breaking changes, plus we have a couple custom fixes applied to the linked branch).